lib/Switch.js

```diff
@@ -143,11 +142,11 @@ export class Switch extends Component {
 							{ transform: [{ translateX: transformSwitch }], width: circleSize * 2 },
 						]}
 					>
-						<Text style={[styles.text, styles.inactiveText, styles.paddingRight, activeTextStyle]}>
+						<Text style={[styles.text, styles.paddingRight, activeTextStyle]}>
 							{activeText}
 						</Text>
 						<Animated.View style={[styles.circle, { backgroundColor: interpolatedCircleColor, width: circleSize, height: circleSize, borderRadius: circleSize / 2 }]} />
-						<Text style={[styles.text, styles.active, styles.paddingLeft, inactiveTextStyle]}>
```

Comment:
Can you please explain this change ?

Reply:
Just removing `styles.active` which isn't defined in the StyleSheet object.